### PR TITLE
Nstonehouse/breakup css grid

### PR DESCRIPTION
**Pull request by njostonehouse**

### CODE REVIEW

File: src/d2l-my-courses-content/d2l-my-courses-content-behavior.js [hunk 1]
```diff
@@ -220,6 +211,15 @@ D2L.MyCourses.MyCoursesContentBehaviorImpl = {
 			&& this._numberOfEnrollments !== 0
 			&& this._enrollments.length === 0;
 	},
+	_getTileGrid: function() {
+		return this.$$('.course-tile-grid');
```
Comment (njostonehouse) [inline src/d2l-my-courses-content/d2l-my-courses-content-behavior.js]: Only `-animated` has `cssGridView` set to `false`. Override the tile grid selector there.

File: src/d2l-my-courses-content/d2l-my-courses-content-behavior.js [hunk 2]
```diff
@@ -220,6 +211,15 @@ D2L.MyCourses.MyCoursesContentBehaviorImpl = {
...
+	},
+	_refreshTileGridImages: function() {
```
Comment (njostonehouse) [inline src/d2l-my-courses-content/d2l-my-courses-content-behavior.js]: Similarly, `CssGridView == true` iterates through the tiles to refresh each image, whereas `== false` has its own technique (overriden in `-animated`).

File: src/d2l-my-courses-content/d2l-my-courses-content-animated.js [hunk 3]
```diff
@@ -267,6 +267,14 @@ Polymer({
 		return false;
 	},
 
+	_getTileGrid: function() {
```
Comment (njostonehouse) [inline src/d2l-my-courses-content/d2l-my-courses-content-animated.js]: Overrides for `CssGridView == true` case.

**Comment by edwardg8** [comment]
Looks all good, there is one thing I hate (i know it was from the old code) is the property name `cssGridView`, `hasCssGridView` may be a better name, but we are ok to leave it I think

**Comment by njostonehouse** [comment]
This is part of a larger effort to get rid of `cssGridView`. It removes it from the `-content` components at least. Then I plan to rename it to `card-grid`.

**Pull request merged**